src/monthlychallenges/apr-2021/index.njk

```diff
@@ -5,6 +5,11 @@ layout: layouts/simple.njk
 date: 2021-04-01
 tags: monthlychallenges
 ---
+
+<div class="alert alert-success" role="alert">
```

Comment:
@teezzan I know you used the same markup as the older pages, but let's make this change:

Let's remove `role="alert"` from all of these (and the other existing ones). The [`alert` role](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Alert_Role) is meant for content that has popped up or changed, not really for static content: 

> The alert role should read out content that has changed, or bring the user's attention to it immediately, so it should not be used for static content or used regularly. Alerts, by definition, are disruptive. Lots of alerts at once or unnecessary alerts will create a bad user experience.  

Maybe while we're at it we can remove it from the older pages too!